RELEASES.md

```diff
@@ -0,0 +1,36 @@
+# CITGM Release Process
+
+This document describes the technical aspects of the citgm release process. The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.
```

Comment:
>The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.

Is this `@nodejs/citgm` as below? If so the whole sentence seems a bit redundant.